agents/kubevirt/fence_kubevirt.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/python -tt
+
+import os
+import sys, re
+import logging
+import atexit
+sys.path.append("/usr/share/fence/")
+from fencing import *
+from fencing import fail, fail_usage, run_delay, EC_STATUS, SyslogLibHandler
+from kubernetes import client, config
```

Comment:
You might need to add try: / except ImportError: pass here and on the config/connect line below to be able to show metadata during compilation (for manpage generation and to pass CI gating).

Run `make xml-upload` and `git add` it to the commit, as it will fail CI gating if we dont have the metadata to compare against.

Reply:
@oalbrigt I added the try/except block, but can you please elaborate about the `make xml-upload`? how do I run it? I couldn't run it in the repository